client/my-sites/plugins/index.js

```diff
@@ -35,6 +36,16 @@ module.exports = function() {
 	}
 
 	if ( config.isEnabled( 'manage/plugins' ) ) {
+		page( '/plugins/wpcom-masterbar-redirect/:site', context => {
+			analytics.tracks.recordEvent( 'calypso_wpcom_masterbar_plugins_view_redirect' );
```

Comment:
Thanks for the suggestions @roundhill and @gwwar! Updated in 068b76a9c656281532bc5c6f548e0042aac4c818.